### ISSUE TITLE
module 'cv2' has no attribute 'VideoCapture'

### ISSUE THREAD

**Issue by dkamajix** [Kosinkadink/ComfyUI-VideoHelperSuite#304]
Hi! Im having some sort of new conflict rising from the CV2 module.
honestly its a bit beyond what I understand.. might be conflicting with something, 
or a recent installation of a node messed something up.
Can anyone here pls help! here is the report:

# ComfyUI Error Report
## Error Details
- **Node Type:** VHS_LoadVideo
- **Exception Type:** AttributeError
- **Exception Message:** module 'cv2' has no attribute 'VideoCapture'
## Stack Trace
```
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 323, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 198, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 169, in _map_node_over_list
    process_inputs(input_dict, i)

  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 158, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 271, in load_video
    return load_video_cv(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^

  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 143, in load_video_cv
    (width, height, fps, duration, total_frames, target_frame_time) = next(gen)
                                                                      ^^^^^^^^^

  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 42, in cv_frame_generator
    video_cap = cv2.VideoCapture(strip_path(video))
                ^^^^^^^^^^^^^^^^

```
## System Information
- **ComfyUI Version:** v0.2.3-10-gc695c4a
- **Arguments:** ComfyUI\main.py --windows-standalone-build
- **OS:** nt
- **Python Version:** 3.11.6 (tags/v3.11.6:8b6ee5b, Oct  2 2023, 14:57:12) [MSC v.1935 64 bit (AMD64)]
- **Embedded Python:** true
- **PyTorch Version:** 2.3.1+cu121
## Devices

- **Name:** cuda:0 NVIDIA GeForce RTX 4080 : cudaMallocAsync
  - **Type:** cuda
  - **VRAM Total:** 17170825216
  - **VRAM Free:** 15745088616
  - **Torch VRAM Total:** 33554432
  - **Torch VRAM Free:** 12254312

## Logs
```
2024-10-16 11:04:02,633 - root - INFO - Total VRAM 16375 MB, total RAM 131014 MB
2024-10-16 11:04:02,634 - root - INFO - pytorch version: 2.3.1+cu121
2024-10-16 11:04:02,634 - root - INFO - Set vram state to: NORMAL_VRAM
2024-10-16 11:04:02,634 - root - INFO - Device: cuda:0 NVIDIA GeForce RTX 4080 : cudaMallocAsync
2024-10-16 11:04:03,577 - root - INFO - Using pytorch cross attention
2024-10-16 11:04:04,644 - root - INFO - [Prompt Server] web root: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\web
2024-10-16 11:04:04,646 - root - INFO - Adding extra search path checkpoints G:\WORKSPACE_AI\00_SD_MODELS\CHECKPOINTS
2024-10-16 11:04:04,646 - root - INFO - Adding extra search path configs G:\WORKSPACE_AI\00_SD_MODELS\models/Stable-diffusion
2024-10-16 11:04:04,646 - root - INFO - Adding extra search path vae G:\WORKSPACE_AI\00_SD_MODELS\VAE
2024-10-16 11:04:04,646 - root - INFO - Adding extra search path loras G:\WORKSPACE_AI\00_SD_MODELS\LORAS
2024-10-16 11:04:04,646 - root - INFO - Adding extra search path loras G:\WORKSPACE_AI\00_SD_MODELS\models/LyCORIS
2024-10-16 11:04:04,647 - root - INFO - Adding extra search path upscale_models G:\WORKSPACE_AI\00_SD_MODELS\ERSGAN
2024-10-16 11:04:04,647 - root - INFO - Adding extra search path upscale_models G:\WORKSPACE_AI\00_SD_MODELS\models/RealESRGAN
2024-10-16 11:04:04,648 - root - INFO - Adding extra search path upscale_models G:\WORKSPACE_AI\00_SD_MODELS\models/SwinIR
2024-10-16 11:04:04,648 - root - INFO - Adding extra search path embeddings G:\WORKSPACE_AI\00_SD_MODELS\EMBEDDINGS
2024-10-16 11:04:04,649 - root - INFO - Adding extra search path hypernetworks G:\WORKSPACE_AI\00_SD_MODELS\models/hypernetworks
2024-10-16 11:04:04,649 - root - INFO - Adding extra search path controlnet G:\WORKSPACE_AI\00_SD_MODELS\CONTROLNET
2024-10-16 11:04:04,649 - root - INFO - Adding extra search path ipadapter G:\WORKSPACE_AI\00_SD_MODELS\IPADAPTERS
2024-10-16 11:04:05,723 - root - WARNING - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\nodes.py", line 2001, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 936, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1073, in get_code
  File "<frozen importlib._bootstrap_external>", line 1130, in get_data
FileNotFoundError: [Errno 2] No such file or directory: 'G:\\WORKSPACE_AI\\02_COMFY\\ComfyUI_windows_portable\\ComfyUI\\custom_nodes\\BiRefNet\\__init__.py'

2024-10-16 11:04:05,724 - root - WARNING - Cannot import G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\BiRefNet module for custom nodes: [Errno 2] No such file or directory: 'G:\\WORKSPACE_AI\\02_COMFY\\ComfyUI_windows_portable\\ComfyUI\\custom_nodes\\BiRefNet\\__init__.py'
2024-10-16 11:04:06,180 - root - WARNING - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\nodes.py", line 2001, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Allor\__init__.py", line 12, in <module>
    NODE_CLASS_MAPPINGS = loader.get_modules()
                          ^^^^^^^^^^^^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Allor\Loader.py", line 348, in get_modules
    from .modules import ImageSegmentation
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Allor\modules\ImageSegmentation.py", line 3, in <module>
    from rembg import remove, new_session
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\rembg\__init__.py", line 5, in <module>
    from .bg import remove
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\rembg\bg.py", line 6, in <module>
    from cv2 import (
ImportError: cannot import name 'BORDER_DEFAULT' from 'cv2' (unknown location)

2024-10-16 11:04:06,181 - root - WARNING - Cannot import G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Allor module for custom nodes: cannot import name 'BORDER_DEFAULT' from 'cv2' (unknown location)
2024-10-16 11:04:06,762 - root - ERROR - [Impact Pack] Failed to import due to several dependencies are missing!!!!
2024-10-16 11:04:07,387 - root - WARNING - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\nodes.py", line 2001, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BiRefNet\__init__.py", line 3, in <module>
    from .BiRefNet_node import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BiRefNet\BiRefNet_node.py", line 7, in <module>
    from models.baseline import BiRefNet
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BiRefNet\models\baseline.py", line 11, in <module>
    from models.backbones.build_backbone import build_backbone
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BiRefNet\models\backbones\build_backbone.py", line 8, in <module>
    from utils import check_download_model
ImportError: cannot import name 'check_download_model' from 'utils' (G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\utils\__init__.py)

2024-10-16 11:04:07,387 - root - WARNING - Cannot import G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BiRefNet module for custom nodes: cannot import name 'check_download_model' from 'utils' (G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\utils\__init__.py)
2024-10-16 11:04:07,391 - root - WARNING - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\nodes.py", line 2001, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BiRefNet-ZHO\__init__.py", line 1, in <module>
    from . import birefnet
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BiRefNet-ZHO\birefnet.py", line 7, in <module>
    from models.baseline import BiRefNet
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BiRefNet\models\baseline.py", line 11, in <module>
    from models.backbones.build_backbone import build_backbone
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BiRefNet\models\backbones\build_backbone.py", line 8, in <module>
    from utils import check_download_model
ImportError: cannot import name 'check_download_model' from 'utils' (G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\utils\__init__.py)

2024-10-16 11:04:07,391 - root - WARNING - Cannot import G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BiRefNet-ZHO module for custom nodes: cannot import name 'check_download_model' from 'utils' (G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\utils\__init__.py)
2024-10-16 11:04:07,402 - ComfyUI-CogVideoXWrapper.custom_cogvideox_transformer_3d - INFO - sageattn not found, using sdpa
2024-10-16 11:04:07,410 - ComfyUI-CogVideoXWrapper.cogvideox_fun.transformer_3d - INFO - sageattn not found, using sdpa
2024-10-16 11:04:07,411 - ComfyUI-CogVideoXWrapper.cogvideox_fun.fun_pab_transformer_3d - INFO - sageattn not found, using sdpa
2024-10-16 11:04:07,485 - dinov2 - WARNING - xFormers not available
2024-10-16 11:04:07,487 - dinov2 - WARNING - xFormers not available
2024-10-16 11:04:08,162 - root - ERROR - [Impact Pack] Failed to import due to several dependencies are missing!!!!
2024-10-16 11:04:08,164 - root - WARNING - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\nodes.py", line 2001, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\__init__.py", line 53, in <module>
    raise e
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\__init__.py", line 31, in <module>
    import impact.subpack_nodes  # This import must be done before cv2.
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\impact_subpack\impact\subpack_nodes.py", line 4, in <module>
    import impact.subcore as subcore
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\impact_subpack\impact\subcore.py", line 17, in <module>
    raise e
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack\impact_subpack\impact\subcore.py", line 13, in <module>
    from ultralytics import YOLO
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\ultralytics\__init__.py", line 10, in <module>
    from ultralytics.data.explorer.explorer import Explorer
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\ultralytics\data\__init__.py", line 3, in <module>
    from .base import BaseDataset
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\ultralytics\data\base.py", line 17, in <module>
    from ultralytics.data.utils import FORMATS_HELP_MSG, HELP_URL, IMG_FORMATS
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\ultralytics\data\utils.py", line 19, in <module>
    from ultralytics.nn.autobackend import check_class_names
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\ultralytics\nn\__init__.py", line 3, in <module>
    from .tasks import (
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\ultralytics\nn\tasks.py", line 10, in <module>
    from ultralytics.nn.modules import (
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\ultralytics\nn\modules\__init__.py", line 20, in <module>
    from .block import (
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\ultralytics\nn\modules\block.py", line 8, in <module>
    from ultralytics.utils.torch_utils import fuse_conv_and_bn
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\ultralytics\utils\__init__.py", line 108, in <module>
    cv2.setNumThreads(0)  # prevent OpenCV from multithreading (incompatible with PyTorch DataLoader)
    ^^^^^^^^^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'setNumThreads'

2024-10-16 11:04:08,165 - root - WARNING - Cannot import G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack module for custom nodes: module 'cv2' has no attribute 'setNumThreads'
2024-10-16 11:04:08,167 - root - WARNING - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\nodes.py", line 2001, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Inference-Core-Nodes\__init__.py", line 1, in <module>
    from inference_core_nodes import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
ModuleNotFoundError: No module named 'inference_core_nodes'

2024-10-16 11:04:08,168 - root - WARNING - Cannot import G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Inference-Core-Nodes module for custom nodes: No module named 'inference_core_nodes'
2024-10-16 11:04:08,239 - root - INFO - Total VRAM 16375 MB, total RAM 131014 MB
2024-10-16 11:04:08,239 - root - INFO - pytorch version: 2.3.1+cu121
2024-10-16 11:04:08,241 - root - INFO - Set vram state to: NORMAL_VRAM
2024-10-16 11:04:08,241 - root - INFO - Device: cuda:0 NVIDIA GeForce RTX 4080 : cudaMallocAsync
2024-10-16 11:04:08,307 - root - WARNING - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\nodes.py", line 2001, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-liveportrait\__init__.py", line 1, in <module>
    from .nodes.live_portrait import LivePortraitNode,FaceCropInfo,Retargeting,LivePortraitVideoNode
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-liveportrait\nodes\live_portrait.py", line 32, in <module>
    from .LivePortrait.src.live_portrait_pipeline import LivePortraitPipeline
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-liveportrait\nodes\LivePortrait\src\live_portrait_pipeline.py", line 20, in <module>
    from .utils.cropper import Cropper
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-liveportrait\nodes\LivePortrait\src\utils\cropper.py", line 12, in <module>
    cv2.setNumThreads(0); cv2.ocl.setUseOpenCL(False)
    ^^^^^^^^^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'setNumThreads'

2024-10-16 11:04:08,307 - root - WARNING - Cannot import G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-liveportrait module for custom nodes: module 'cv2' has no attribute 'setNumThreads'
2024-10-16 11:04:08,318 - ComfyUI-LivePortraitKJ.nodes - WARNING - Can't load MediaPipe, MediaPipeCropper not available
2024-10-16 11:04:08,319 - ComfyUI-LivePortraitKJ.nodes - WARNING - Can't load MediaPipe, MediaPipeCropper not available
2024-10-16 11:04:08,320 - ComfyUI-LivePortraitKJ.nodes - WARNING - Can't load FaceAlignment, CropperFaceAlignment not available
2024-10-16 11:04:08,325 - root - WARNING - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\nodes.py", line 2001, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ\__init__.py", line 1, in <module>
    from .nodes import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ\nodes.py", line 42, in <module>
    from .liveportrait.utils.crop import _transform_img_kornia
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ\liveportrait\utils\crop.py", line 11, in <module>
    CV2_INTERP = cv2.INTER_LINEAR
                 ^^^^^^^^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'INTER_LINEAR'

2024-10-16 11:04:08,326 - root - WARNING - Cannot import G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ module for custom nodes: module 'cv2' has no attribute 'INTER_LINEAR'
2024-10-16 11:04:08,328 - root - WARNING - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\nodes.py", line 2001, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ-MPS\__init__.py", line 1, in <module>
    from .nodes import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ-MPS\nodes.py", line 8, in <module>
    assert torch.backends.mps.is_available(), "You are trying to use a Apple Silicon node on a Windows or Linux device. Download the original node from https://github.com/kijai/ComfyUI-LivePortraitKJ."
AssertionError: You are trying to use a Apple Silicon node on a Windows or Linux device. Download the original node from https://github.com/kijai/ComfyUI-LivePortraitKJ.

2024-10-16 11:04:08,328 - root - WARNING - Cannot import G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ-MPS module for custom nodes: You are trying to use a Apple Silicon node on a Windows or Linux device. Download the original node from https://github.com/kijai/ComfyUI-LivePortraitKJ.
2024-10-16 11:04:09,197 - root - INFO - --------------
2024-10-16 11:04:09,197 - root - INFO - [91m ### Mixlab Nodes: [93mLoaded
2024-10-16 11:04:09,205 - root - INFO - ChatGPT.available True
2024-10-16 11:04:09,206 - root - INFO - edit_mask.available True
2024-10-16 11:04:09,210 - root - INFO - LaMaInpainting.available False
2024-10-16 11:04:09,327 - root - INFO - ClipInterrogator.available True
2024-10-16 11:04:09,329 - root - INFO - PromptGenerate.available True
2024-10-16 11:04:09,329 - root - INFO - ChinesePrompt.available True
2024-10-16 11:04:09,329 - root - INFO - RembgNode_.available False
2024-10-16 11:04:09,564 - root - INFO - TripoSR.available
2024-10-16 11:04:09,565 - root - INFO - MiniCPMNode.available
2024-10-16 11:04:09,587 - root - INFO - Scenedetect.available False
2024-10-16 11:04:09,665 - root - INFO - FishSpeech.available
2024-10-16 11:04:09,673 - root - INFO - SenseVoice.available
2024-10-16 11:04:09,694 - root - INFO - Whisper.available False
2024-10-16 11:04:09,702 - root - INFO - FalVideo.available
2024-10-16 11:04:09,702 - root - INFO - [93m -------------- [0m
2024-10-16 11:04:10,237 - numexpr.utils - INFO - Note: NumExpr detected 32 cores but "NUMEXPR_MAX_THREADS" not set, so enforcing safe limit of 8.
2024-10-16 11:04:10,797 - root - WARNING - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\nodes.py", line 2001, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-reactor-node\__init__.py", line 23, in <module>
    from .nodes import NODE_CLASS_MAPPINGS, NODE_DISPLAY_NAME_MAPPINGS
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-reactor-node\nodes.py", line 15, in <module>
    from insightface.app.common import Face
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\insightface\__init__.py", line 18, in <module>
    from . import app
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\insightface\app\__init__.py", line 2, in <module>
    from .mask_renderer import *
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\insightface\app\mask_renderer.py", line 4, in <module>
    import albumentations as A
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\albumentations\__init__.py", line 6, in <module>
    from .augmentations import *
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\albumentations\augmentations\__init__.py", line 1, in <module>
    from .blur.functional import *
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\albumentations\augmentations\blur\__init__.py", line 1, in <module>
    from .functional import *
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\albumentations\augmentations\blur\functional.py", line 9, in <module>
    from albucore import clipped, float32_io, maybe_process_in_chunks, preserve_channel_dim
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\albucore\__init__.py", line 3, in <module>
    from .decorators import *
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\albucore\decorators.py", line 7, in <module>
    from albucore.utils import MONO_CHANNEL_DIMENSIONS, NUM_MULTI_CHANNEL_DIMENSIONS, P
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\albucore\utils.py", line 44, in <module>
    np.uint8: cv2.CV_8U,
              ^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'CV_8U'

2024-10-16 11:04:10,798 - root - WARNING - Cannot import G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-reactor-node module for custom nodes: module 'cv2' has no attribute 'CV_8U'
2024-10-16 11:04:10,813 - py.warnings - WARNING - G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-segment-anything-2\sam2\modeling\sam\transformer.py:20: UserWarning: Flash Attention is disabled as it requires a GPU with Ampere (8.0) CUDA capability.
  OLD_GPU, USE_FLASH_ATTN, MATH_KERNEL_ON = get_sdpa_settings()

2024-10-16 11:04:11,112 - py.warnings - WARNING - G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\pkg_resources\__init__.py:123: PkgResourcesDeprecationWarning: py-cuda11x is an invalid version and will not be supported in a future release
  warnings.warn(

2024-10-16 11:04:12,167 - root - WARNING - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\nodes.py", line 2001, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-YoloWorld-EfficientSAM\__init__.py", line 1, in <module>
    from . import YOLO_WORLD_EfficientSAM
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-YoloWorld-EfficientSAM\YOLO_WORLD_EfficientSAM.py", line 6, in <module>
    import supervision as sv
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\supervision\__init__.py", line 9, in <module>
    from supervision.annotators.core import (
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\supervision\annotators\core.py", line 19, in <module>
    from supervision.draw.utils import draw_polygon
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\python_embeded\Lib\site-packages\supervision\draw\utils.py", line 164, in <module>
    text_font: int = cv2.FONT_HERSHEY_SIMPLEX,
                     ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'FONT_HERSHEY_SIMPLEX'

2024-10-16 11:04:12,168 - root - WARNING - Cannot import G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-YoloWorld-EfficientSAM module for custom nodes: module 'cv2' has no attribute 'FONT_HERSHEY_SIMPLEX'
2024-10-16 11:04:18,437 - root - WARNING - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\nodes.py", line 2001, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 936, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1073, in get_code
  File "<frozen importlib._bootstrap_external>", line 1130, in get_data
FileNotFoundError: [Errno 2] No such file or directory: 'G:\\WORKSPACE_AI\\02_COMFY\\ComfyUI_windows_portable\\ComfyUI\\custom_nodes\\reactor\\__init__.py'

2024-10-16 11:04:18,438 - root - WARNING - Cannot import G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\reactor module for custom nodes: [Errno 2] No such file or directory: 'G:\\WORKSPACE_AI\\02_COMFY\\ComfyUI_windows_portable\\ComfyUI\\custom_nodes\\reactor\\__init__.py'
2024-10-16 11:04:20,899 - root - WARNING - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\nodes.py", line 2001, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\was-node-suite-comfyui\__init__.py", line 1, in <module>
    from .WAS_Node_Suite import NODE_CLASS_MAPPINGS
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\was-node-suite-comfyui\WAS_Node_Suite.py", line 14511, in <module>
    build_info = ' '.join(cv2.getBuildInformation().split())
                          ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'getBuildInformation'

2024-10-16 11:04:20,899 - root - WARNING - Cannot import G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\was-node-suite-comfyui module for custom nodes: module 'cv2' has no attribute 'getBuildInformation'
2024-10-16 11:04:20,906 - root - INFO - 
Import times for custom nodes:
2024-10-16 11:04:20,906 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\SDXLAspectRatio.py
2024-10-16 11:04:20,907 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\highresfix.py
2024-10-16 11:04:20,907 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Image-Selector
2024-10-16 11:04:20,907 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\websocket_image_save.py
2024-10-16 11:04:20,908 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\canvas_tab
2024-10-16 11:04:20,908 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-SDXL-EmptyLatentImage
2024-10-16 11:04:20,908 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\LoadLoraWithTags
2024-10-16 11:04:20,909 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-CustomScheduler
2024-10-16 11:04:20,909 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\Winston
2024-10-16 11:04:20,910 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\sdxl_prompt_styler
2024-10-16 11:04:20,910 - root - INFO -    0.0 seconds (IMPORT FAILED): G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\reactor
2024-10-16 11:04:20,910 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\extended-saveimage-comfyui
2024-10-16 11:04:20,911 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-RAVE
2024-10-16 11:04:20,911 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_Ib_CustomNodes
2024-10-16 11:04:20,911 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-sampler-lcm-alternative
2024-10-16 11:04:20,912 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\cg-use-everywhere
2024-10-16 11:04:20,912 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_IPAdapter_plus
2024-10-16 11:04:20,912 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_SUNoise
2024-10-16 11:04:20,913 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_Noise
2024-10-16 11:04:20,913 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\sd-dynamic-thresholding
2024-10-16 11:04:20,913 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\efficiency-nodes-comfyui
2024-10-16 11:04:20,914 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Logic
2024-10-16 11:04:20,914 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\klinter_nodes
2024-10-16 11:04:20,914 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_Cutoff
2024-10-16 11:04:20,915 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite
2024-10-16 11:04:20,917 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_ADV_CLIP_emb
2024-10-16 11:04:20,918 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUi_NNLatentUpscale
2024-10-16 11:04:20,918 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\stability-ComfyUI-nodes
2024-10-16 11:04:20,918 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\cg-image-picker
2024-10-16 11:04:20,919 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\cg_custom_core
2024-10-16 11:04:20,919 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\masquerade-nodes-comfyui
2024-10-16 11:04:20,919 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\FreeU_Advanced
2024-10-16 11:04:20,920 - root - INFO -    0.0 seconds (IMPORT FAILED): G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ-MPS
2024-10-16 11:04:20,920 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyLiterals
2024-10-16 11:04:20,920 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-post-processing-nodes
2024-10-16 11:04:20,921 - root - INFO -    0.0 seconds (IMPORT FAILED): G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Inference-Core-Nodes
2024-10-16 11:04:20,921 - root - INFO -    0.0 seconds (IMPORT FAILED): G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\BiRefNet
2024-10-16 11:04:20,921 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-WD14-Tagger
2024-10-16 11:04:20,922 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-ELLA
2024-10-16 11:04:20,922 - root - INFO -    0.0 seconds (IMPORT FAILED): G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BiRefNet-ZHO
2024-10-16 11:04:20,922 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_experiments
2024-10-16 11:04:20,923 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_JPS-Nodes
2024-10-16 11:04:20,923 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-SuperBeasts
2024-10-16 11:04:20,923 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\PowerNoiseSuite
2024-10-16 11:04:20,924 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ymc-node-suite-comfyui
2024-10-16 11:04:20,924 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ControlAltAI-Nodes
2024-10-16 11:04:20,924 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux
2024-10-16 11:04:20,925 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-TiledDiffusion
2024-10-16 11:04:20,925 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Video-Matting
2024-10-16 11:04:20,925 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\images-grid-comfy-plugin
2024-10-16 11:04:20,926 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-QualityOfLifeSuit_Omar92
2024-10-16 11:04:20,926 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-animatediff
2024-10-16 11:04:20,927 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\IPAdapter-ComfyUI
2024-10-16 11:04:20,927 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-layerdiffuse
2024-10-16 11:04:20,927 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_FizzNodes
2024-10-16 11:04:20,928 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfy_PoP
2024-10-16 11:04:20,928 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\mikey_nodes
2024-10-16 11:04:20,928 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Extra-Samplers
2024-10-16 11:04:20,929 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-various
2024-10-16 11:04:20,929 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-RefUNet
2024-10-16 11:04:20,929 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyMath
2024-10-16 11:04:20,930 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_essentials
2024-10-16 11:04:20,930 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Florence2
2024-10-16 11:04:20,930 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Custom-Scripts
2024-10-16 11:04:20,931 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_UltimateSDUpscale
2024-10-16 11:04:20,933 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfy-image-saver
2024-10-16 11:04:20,934 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\virtuoso-nodes
2024-10-16 11:04:20,934 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Frame-Interpolation
2024-10-16 11:04:20,934 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-DepthAnythingV2
2024-10-16 11:04:20,935 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\save-image-extended-comfyui
2024-10-16 11:04:20,935 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\Derfuu_ComfyUI_ModdedNodes
2024-10-16 11:04:20,935 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-GGUF
2024-10-16 11:04:20,936 - root - INFO -    0.0 seconds (IMPORT FAILED): G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack
2024-10-16 11:04:20,936 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-JDCN
2024-10-16 11:04:20,936 - root - INFO -    0.0 seconds (IMPORT FAILED): G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-YoloWorld-EfficientSAM
2024-10-16 11:04:20,937 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\rgthree-comfy
2024-10-16 11:04:20,937 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_ExtraModels
2024-10-16 11:04:20,937 - root - INFO -    0.0 seconds (IMPORT FAILED): G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Allor
2024-10-16 11:04:20,938 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\OneButtonPrompt
2024-10-16 11:04:20,938 - root - INFO -    0.0 seconds (IMPORT FAILED): G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-LivePortraitKJ
2024-10-16 11:04:20,938 - root - INFO -    0.0 seconds (IMPORT FAILED): G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-reactor-node
2024-10-16 11:04:20,939 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-CogVideoXWrapper
2024-10-16 11:04:20,939 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_bitsandbytes_NF4
2024-10-16 11:04:20,939 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-KJNodes
2024-10-16 11:04:20,940 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-AnimateDiff-Evolved
2024-10-16 11:04:20,940 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_Comfyroll_CustomNodes
2024-10-16 11:04:20,940 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-dream-project
2024-10-16 11:04:20,941 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\facerestore_cf
2024-10-16 11:04:20,941 - root - INFO -    0.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Inspire-Pack
2024-10-16 11:04:20,941 - root - INFO -    0.0 seconds (IMPORT FAILED): G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-liveportrait
2024-10-16 11:04:20,942 - root - INFO -    0.1 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Crystools
2024-10-16 11:04:20,942 - root - INFO -    0.1 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_tinyterraNodes
2024-10-16 11:04:20,942 - root - INFO -    0.1 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfy_mtb
2024-10-16 11:04:20,943 - root - INFO -    0.1 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-segment-anything-2
2024-10-16 11:04:20,943 - root - INFO -    0.1 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-FollowYourEmojiWrapper
2024-10-16 11:04:20,943 - root - INFO -    0.1 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Advanced-ControlNet
2024-10-16 11:04:20,944 - root - INFO -    0.1 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\sdxl_prompt_styler-main
2024-10-16 11:04:20,944 - root - INFO -    0.1 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-dynamicprompts
2024-10-16 11:04:20,944 - root - INFO -    0.2 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-ollama
2024-10-16 11:04:20,945 - root - INFO -    0.2 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-N-Nodes
2024-10-16 11:04:20,945 - root - INFO -    0.3 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-ADMotionDirector
2024-10-16 11:04:20,945 - root - INFO -    0.4 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_segment_anything
2024-10-16 11:04:20,946 - root - INFO -    0.4 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Easy-Use
2024-10-16 11:04:20,946 - root - INFO -    0.4 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager
2024-10-16 11:04:20,946 - root - INFO -    0.5 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\a-person-mask-generator
2024-10-16 11:04:20,947 - root - INFO -    0.6 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-art-venture
2024-10-16 11:04:20,949 - root - INFO -    0.6 seconds (IMPORT FAILED): G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-BiRefNet
2024-10-16 11:04:20,949 - root - INFO -    0.6 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-propost
2024-10-16 11:04:20,950 - root - INFO -    1.0 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-mixlab-nodes
2024-10-16 11:04:20,950 - root - INFO -    1.1 seconds (IMPORT FAILED): G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\was-node-suite-comfyui
2024-10-16 11:04:20,950 - root - INFO -    1.2 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\SeargeSDXL
2024-10-16 11:04:20,951 - root - INFO -    1.3 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-SUPIR
2024-10-16 11:04:20,951 - root - INFO -    5.6 seconds: G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_LLM_party
2024-10-16 11:04:20,951 - root - INFO - 
2024-10-16 11:04:20,966 - root - INFO - Starting server

2024-10-16 11:04:20,966 - root - INFO - To see the GUI go to: http://127.0.0.1:8188
2024-10-16 11:04:27,544 - asyncio - ERROR - Exception in callback _ProactorBasePipeTransport._call_connection_lost(None)
handle: <Handle _ProactorBasePipeTransport._call_connection_lost(None)>
Traceback (most recent call last):
  File "asyncio\events.py", line 80, in _run
  File "asyncio\proactor_events.py", line 165, in _call_connection_lost
ConnectionResetError: [WinError 10054] An existing connection was forcibly closed by the remote host
2024-10-16 11:04:30,742 - root - INFO - got prompt
2024-10-16 11:04:33,228 - root - ERROR - !!! Exception during processing !!! module 'cv2' has no attribute 'VideoCapture'
2024-10-16 11:04:33,229 - root - ERROR - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 323, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 198, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 169, in _map_node_over_list
    process_inputs(input_dict, i)
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 158, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 271, in load_video
    return load_video_cv(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 143, in load_video_cv
    (width, height, fps, duration, total_frames, target_frame_time) = next(gen)
                                                                      ^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 42, in cv_frame_generator
    video_cap = cv2.VideoCapture(strip_path(video))
                ^^^^^^^^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'VideoCapture'

2024-10-16 11:04:33,230 - root - INFO - Prompt executed in 1.03 seconds
2024-10-16 11:07:48,378 - root - INFO - got prompt
2024-10-16 11:07:50,824 - root - ERROR - !!! Exception during processing !!! module 'cv2' has no attribute 'VideoCapture'
2024-10-16 11:07:50,825 - root - ERROR - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 323, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 198, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 169, in _map_node_over_list
    process_inputs(input_dict, i)
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 158, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 271, in load_video
    return load_video_cv(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 143, in load_video_cv
    (width, height, fps, duration, total_frames, target_frame_time) = next(gen)
                                                                      ^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 42, in cv_frame_generator
    video_cap = cv2.VideoCapture(strip_path(video))
                ^^^^^^^^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'VideoCapture'

2024-10-16 11:07:50,826 - root - INFO - Prompt executed in 0.97 seconds
2024-10-16 11:09:51,200 - root - INFO - got prompt
2024-10-16 11:09:54,703 - root - ERROR - !!! Exception during processing !!! module 'cv2' has no attribute 'VideoCapture'
2024-10-16 11:09:54,704 - root - ERROR - Traceback (most recent call last):
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 323, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 198, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 169, in _map_node_over_list
    process_inputs(input_dict, i)
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\execution.py", line 158, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 271, in load_video
    return load_video_cv(**kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 143, in load_video_cv
    (width, height, fps, duration, total_frames, target_frame_time) = next(gen)
                                                                      ^^^^^^^^^
  File "G:\WORKSPACE_AI\02_COMFY\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-VideoHelperSuite\videohelpersuite\load_video_nodes.py", line 42, in cv_frame_generator
    video_cap = cv2.VideoCapture(strip_path(video))
                ^^^^^^^^^^^^^^^^
AttributeError: module 'cv2' has no attribute 'VideoCapture'

2024-10-16 11:09:54,706 - root - INFO - Prompt executed in 1.47 seconds
```
## Attached Workflow
Please make sure that workflow does not contain any sensitive information such as API keys or passwords.
```
Workflow too large. Please manually upload the workflow from local file system.
```

## Additional Context
(Please add any additional context or steps to reproduce the error here)

**Comment by AustinMroz** [comment]
Duplicate of #185.
Th linked issue provides a potential solution [here](https://github.com/Kosinkadink/ComfyUI-VideoHelperSuite/issues/185#issuecomment-2113471677)
